ysh/grammar_gen.py

```diff
@@ -191,24 +192,26 @@ def main(argv):
 
         is_expr = grammar_name in ('calc', 'grammar')
 
-        parse_opts = optview.Parse()
+        parse_opts = optview.Parse([], [])
         parse_ctx = parse_lib.ParseContext(arena, parse_opts, {}, gr)
-        p = expr_parse.ExprParser(parse_ctx, gr)
+        p = expr_parse.ExprParser(parse_ctx, gr, False)
```

Comment:
Thanks for fixing this, I think I used it when starting the pgen2 integration, and then stopped using it

I probably will use it when parsing func and proc signatures